Compiler.cmake

```diff
@@ -11,6 +11,7 @@
 # Input Variables
 # * COMMON_MINIMUM_GCC_VERSION check for a minimum gcc version, default 4.4
 # * COMMON_USE_CXX03 When set, do not enable C++11 language features
+# * COMMON_DISABLE When set, do not set any flags
```

Comment:
@mgeplf: Default is -std=c++11 for gcc 4.8 (which is the swag-ed compiler on Intel, if that still means anything)